Makefile.gen

```diff
@@ -16,8 +16,17 @@
 
 ######################################################################
 
+CANONICAL_GOPATH := $(shell go env GOPATH)/src/kpt.dev/configsync
+
+.PHONY: symlink-gopath
+symlink-gopath:
+	@[[ -d $(CANONICAL_GOPATH) ]] || ln -s $(shell pwd) $(CANONICAL_GOPATH)
+	@[[ $(shell pwd) == $(shell realpath $(CANONICAL_GOPATH)) ]]
```

Comment:
fwiw, i dont think realpath exists on mac by default, but it can be installed.

Reply:
would this exit if not true? or does it just continue?

Reply:
The statement returns a nonzero exit code, causing the make target to error

Reply:
can you make it print an error message on failure? to stderr perhaps

Reply:
done